### ISSUE TITLE
Some of the peer review functions does not work

### ISSUE THREAD

**Issue by neslihanturan** [commons-app/apps-android-commons#3038]
**Summary:** 
This issue is introduced after #2937 is merged. Here is a brief explanation of problems in peer review.

| Operation  | Expectation | Reality |
| ------------- | ------------- | ------------- |
| Is this in scope - No, out of scope | Requests reason -> nominates for deletion with delete reason -> Brings first question with new image | Does nothing |
| Is this in scope - Seems fine  | Brings next question | Brings next question |
--
| Copyright? - No, copy right violation | Requests reason -> nominates for deletion with delete reason -> Brings first question with new image | Brings first question |
| Copyright? - Seems fine  | Brings next question | Brings next question |
--
| Category? - mis-categorized | Requests category check -> Brings first question with new image | Requests category check -> Brings next question |
| Category? - Seems fine  | Brings next question | Brings next question |
--
| Thanks? - yes, why not | Sends thanks -> Brings first question with new image | Brings first question with same image |
| Thanks? - Seems fine  | Brings first question with new image |  Brings first question with same image |
